flux-broker-attributes.adoc didn't make it into distribution tarball

Not sure exactly why but `flux-broker-attributes(7)` manpage source didn't make it into the `make dist` tarball:
```
> tar tf flux-core-0.4.0.tar.gz  | grep man7
flux-core-0.4.0/doc/man7/
flux-core-0.4.0/doc/man7/Makefile.am
flux-core-0.4.0/doc/man7/Makefile.in
flux-core-0.4.0/doc/man7/COPYRIGHT.adoc
```

This is causing `make install` failure when trying to build an RPM:
```
DEBUG:  /usr/bin/install -c -m 644 ./flux-broker-attributes.7 '/builddir/build/BUILDROOT/flux-core-0.4.0-1.ch5.4.x86_64/opt/flux-core/share/man/man7'
DEBUG: /usr/bin/install: cannot stat `./flux-broker-attributes.7': No such file or directory
DEBUG: make[3]: Leaving directory `/builddir/build/BUILD/flux-core-0.4.0/doc/man7'
DEBUG: make[3]: *** [install-man7] Error 1
DEBUG: make[2]: *** [install-am] Error 2
```

--- a/doc/man7/Makefile.am
+++ b/doc/man7/Makefile.am
@@ -1,8 +1,8 @@
 MAN7_FILES = \
 	flux-broker-attributes.7
 
-ADOC_FILES  = $(MAN1_FILES:%.7=%.adoc)
-XML_FILES   = $(MAN1_FILES:%.7=%.xml)
+ADOC_FILES  = $(MAN7_FILES:%.7=%.adoc)
+XML_FILES   = $(MAN7_FILES:%.7=%.xml)
 
 if HAVE_A2X
 dist_man_MANS = $(MAN7_FILES)

